packages/react-sdk/src/components/image.tsx

```diff
@@ -1,24 +1,85 @@
 import React, { forwardRef, type ElementRef, type ComponentProps } from "react";
+import { getImageAttributes, type ImageLoader } from "../component-utils/image";
+
+const getInt = (x: unknown): number | undefined => {
```

Comment:
we have a convention to never use single-char variables like this
```suggestion
const getInt = (value: unknown): number | undefined => {
```

Reply:
IMO in math-like utils short x, y, z, a, b are ok. Sometimes x is just x and has no name or value ;-)

Reply:
in this simple function it doesn't mattter, but we need to follow convention, because the next person will write Xs all over the place and point at your function without context as prior art.

always set an example, always assume people won't understand your special context